libs/execution/src/execution_parameter_callbacks.cpp

```diff
@@ -0,0 +1,73 @@
+//  Copyright (c) 2020 ETH Zurich
+//  Copyright (c) 2017 Hartmut Kaiser
+//  Copyright (c) 2017 Google
+//
+//  SPDX-License-Identifier: BSL-1.0
+//  Distributed under the Boost Software License, Version 1.0. (See accompanying
+//  file LICENSE_1_0.txt or copy at http://www.boost.org/LICENSE_1_0.txt)
+
+#include <hpx/config.hpp>
+#include <hpx/execution/detail/execution_parameter_callbacks.hpp>
+
+#include <cstddef>
+#include <type_traits>
+#include <utility>
+
+///////////////////////////////////////////////////////////////////////////////
+namespace hpx { namespace parallel { namespace execution { namespace detail {
+    static get_os_thread_count_type get_os_thread_count_f;
```

Comment:
May I suggest to hide these static variables as function local statics to avoid initialization sequencing problems?